NDB.Covid19/NDB.Covid19.Droid/Resources/layout/registered_page.xml

```diff
@@ -1,150 +1,146 @@
-            android:text="Dummy text"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+        android:id="@+id/registered_title"
+        style="@style/HeaderText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/defaultHorizontalMargin"
+        android:layout_marginVertical="@dimen/defaultVerticalMargin"
+        android:paddingBottom="@dimen/defaultVerticalMargin"
+        android:text="Dummy text"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/close_cross_btn" />
+
     <ScrollView
-            android:orientation="vertical"
-            android:layout_width="match_parent"
-            android:layout_height="0dp"
-            android:layout_marginStart="24dp"
-            android:layout_marginEnd="24dp"
-            android:layout_marginBottom="24dp"
-            app:layout_constraintBottom_toTopOf="@id/registered_button"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/registered_title">
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_marginHorizontal="@dimen/defaultHorizontalMargin"
+        android:layout_marginBottom="@dimen/defaultVerticalMargin"
+        android:orientation="vertical"
+        app:layout_constraintBottom_toTopOf="@id/registered_button"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/registered_title">
+
         <LinearLayout
-                android:id="@+id/registered_content"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:orientation="vertical"
-        >
+            android:id="@+id/registered_content"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
 
             <TextView
-                    android:id="@+id/registered_tick_text"
-                    style="@style/PrimaryText"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:drawableStart="@drawable/check_shield"
-                    android:drawablePadding="19dp"
-                    android:gravity="center_vertical"
-                    android:text="Dummy text" />
+                android:id="@+id/registered_tick_text"
+                style="@style/PrimaryText"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:drawableStart="@drawable/check_shield"
+                android:drawablePadding="19dp"
+                android:gravity="center_vertical"
+                android:text="Dummy text" />
```

Comment:
?

Reply:
exactly how it states ;) dummy text to see how it presents in the view in Layout designer where we have no access to localization strings. normally on Android it can be done in nicer way as there are xml files for localizations but as we use xamarin we care limited to such hardcoded strings. it is overwritten immediately in OnCreate method for activity